package.json

```diff
@@ -1,9 +1,21 @@
 {
   "name": "opg-sirius-user-management",
   "version": "0.1.0",
+  "license": "MIT",
+  "scripts": {
+    "build": "webpack"
+  },
   "dependencies": {
     "@ministryofjustice/frontend": "^0.0.19-alpha",
-    "govuk-frontend": "^3.9.0",
+    "govuk-frontend": "^3.9.0"
```

Comment:
Aren't these also devDependencies since they never get deployed, but are used to produce the thing that gets deployed?

Reply:
It's much of a muchness since we're not publishing an npm package, but I tend to go by the idea that dependencies are shown to users, devDepenecies are for tooling.

The once useful time I've found a distinction is evaluating vulnerabilities: dependencies are higher priority because they'll directly affect users. A devDependencies vulnerability is often reliant on such obscure misconfiguration that it's not a practical threat at all.

Reply:
That seems a reasonable reason. I think my main issue is that I always forget to pass in `-D` when installing packages so prefer the easy life of everything in `dependencies`...